app/models/webhook.rb

```diff
@@ -53,16 +53,18 @@ def disable!
   end
 
   def required_permissions
-    events.map { |event| Webhook.permission_for_event(event) }
+    events
+      .map { |event| Webhook.permission_for_event(event) }
+      .uniq
   end
 
   def self.permission_for_event(event)
```

Comment:
I think we might be able to change this to be simpler, as it's only used in the view:

```
Webhook::EVENTS.filter { |event| !current_user.role.can?(Webhook.permission_for_event(event)) }
```

So like, we could have `Webhook.permissions_for(current_user)` which would return the webhook events the current user could create based on role